### ISSUE TITLE
Simple Mobile Tools keep disappearing after first run after install

### ISSUE THREAD

**Issue by kurzkopfgleitbeutler** [SimpleMobileTools/Simple-Calendar#824]
Hi, thanks for your work.

Intended Behaviour:
Apps can repeatedly be run.

What happens:
After Install, Apps open once. After closing the app, or opening the launcher a second time, app can't be run a second time:
1. App Entry is not in Launcher anymore and
2. "Open"-Button on Play Store / F-Droid -site of App is not visible anymore
3. App is still listed in "All Apps" system settings 

Details:
Android 8.1.0 on Google Nexus 6P
Apps installed via Play Store and F-Droid

Wall of Text:
Simple Calendar is one of the most important apps on my phone, and I used it to have a 100% "store locally only"-events calendar. Thankfully I made a backup of it today before trying to upgrade to the pro version.
I had installed a pre-pro version from F-Droid, and alongside it, the pro version as soon as it came out, from the Play Store.
The Play Store version had this problem, which is why I continued using the old F-Droid version.
However, today I uninstalled the old version after re-installing the pro version, in the hopes that it would Just Work Out™.
Now, no version of Simple Calendar is usable on my device anymore, because this problem happens for all of them, no matter how often I reboot or un- and re-install (both from Play Store and F-Droid).

While this has happened with several of the Simple Mobile Tools Apps, I've still got the old versions of the others and therefore at the moment no intention of uninstalling them.

Possibly Related?: https://android.stackexchange.com/questions/59270/app-icons-disappear-from-home-screen-on-app-update

Please ask for more information if necessary.

**Comment by tibbi** [comment]
Hey, it will most likely be caused by your launcher not handling icon customization properly. Do you maybe use Simple Thank You with some shared theme enabled? If you do, just disable it and stick to the default orange icon color, or try a different launcher. You should still be able to launch the calendar via a widget. Thanks

**Comment by kurzkopfgleitbeutler** [comment]
Sorry I took so long to answer.

I am able to run Simple Calendar through a widget, thank you for pointing out this workaround.

I have Simple Thank You installed, and activated the shared theme in all apps, yes.
I have now deactivated it in all installed apps, and set them to the default icon color
. However, Simple Calendar and Simple Gallery are still affected, while Simple Contacts remains unaffected by the issue.

I have tried with the following app launchers, and none show an Icon for Simple Calendar:
Play Store:
- Google Now Launcher v 1.4.large (no newer version available for my system)
F-Droid:
- HGLauncher v 1.4.1.2
- KISS Launcher v 3.7.9
- OpenLauncher v 0.6.1
- Slim Launcher v 2.2.7
Which leads me to believe that they all use the same underlying system service.

If I can help with further details, I'll be glad to provide them.
